olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff

```diff
@@ -0,0 +1,70 @@
+--- /home/tolusha/gocode/src/github.com/eclipse/che-operator/olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.0/eclipse-che-preview-kubernetes.v7.9.0.clusterserviceversion.yaml	2020-02-26 09:42:38.504787505 +0200
++++ /home/tolusha/gocode/src/github.com/eclipse/che-operator/olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml	2020-03-08 15:51:15.980971338 +0200
+@@ -19,7 +19,7 @@
+                 "cheImageTag": "",
+                 "devfileRegistryImage": "",
+                 "pluginRegistryImage": "",
+-                "tlsSupport": false,
++                "tlsSupport": true,
```

Comment:
So we're turning https on by default in 7.9.1? Are we doing the same in 7.10?

Reply:
sure